packages/@react-aria/dnd/src/useDroppableCollection.ts

```diff
@@ -70,9 +70,9 @@ export function useDroppableCollection(props: DroppableCollectionOptions, state:
     let filteredItems = items;
     if (acceptedDragTypes) {
```

Comment:
I guess this check isn't needed anymore? Or maybe it should be `if (acceptedDragTypes !== 'all' || shouldAcceptItemDrop)` as an optimization?

Reply:
ah yep, thanks for catching this